### PR TITLE
fix: the server watcher

### PR DESCRIPTION
**Pull request by Flower-F**
https://github.com/sanyuan0704/island.js/issues/75

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -158,7 +159,7 @@
   ],
   "lint-staged": {
     "**/*.{js,jsx,tsx,ts,json}": [
-      "eslint --fix"
+      "pnpm run lint:fix"
```
Comment (sanyuan0704) [inline package.json:162]: Why add `pnpm run lint:fix`?

Comment (sanyuan0704) [reply package.json:162]: `eslint --fix` will execute lint for the files changed in git stage.And `pnpm run lint:fix` will run the complete process for all files.

**Comment by sanyuan0704** [comment]
LGTM

**Review by sanyuan0704 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Flower-F) [reply package.json]: Sorry, I just see this [article](https://juejin.cn/book/7050063811973218341/section/7058853948060336163) uses the `npm run lint:script` in lint-staged.

**Pull request merged**